packages/generators/init-generator.ts

```diff
@@ -84,12 +89,15 @@ export default class InitGenerator extends Generator {
 		return this.prompt([Confirm("entryType", "Will your application have multiple bundles?", false)])
 			.then(
 				(entryTypeAnswer: { entryType: boolean }): Promise<{}> => {
```

Comment:
this result var is not stored in `this.configuration`, so I think I'm missing something in the way you're suggesting, how do I check this against `this.configuration`?

Reply:
`checkDefaults(this.config......entry)` after those are set

Reply:
I think I get it, thanks man, sounding neat, trying this out.